asset/css/item-layout.less

```diff
@@ -50,8 +50,10 @@
 
   header {
     display: flex;
-    align-items: flex-start;
+    align-items: baseline;
     justify-content: space-between;
+
+    height: 1.5em;
```

Comment:
This must not be applied to `detailed-item-layout`. The header is allowed to wrap in this case.